style.css

```diff
@@ -0,0 +1,20 @@
+
+
+body {
+  color: white;
+  text-align: left;
+  margin-left: 50px;
+  margin-right: 50px;
+  background-color: black;
+}
+
+h1 {
+  font-family: 'Lato', sans-serif;
+  text-align: center;
+}
+h2 {
```

Comment:
for consistency,  add a line of whitespace between the `{` and the next element like you did on lines 9-10, or remove the line of whitespace on line 10.  I prefer the former, as I think it is easier to read.